torch/nn/functional.py

```diff
@@ -33,6 +33,12 @@
     True``.
     Please see the notes on :doc:`/notes/randomness` for background.
 
+Note:
```

Comment:
Same suggestion to use language like "Supports TensorFloat32" and link the TensorFloat32 text to the existing note.

Reply:
fixed